src/main/java/CinemaRoomRestService/RestService/domain/entity/Statistics.java

```diff
@@ -0,0 +1,25 @@
+package CinemaRoomRestService.RestService.domain.entity;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+@Getter
+public class Statistics {
+    private int currentIncome;
+    private int availableSeats;
+    private int purchasedTickets;
+
+    @Autowired
+    public Statistics(List<Seat> list) {
```

Comment:
You are creating that object in heap every time then you need statistics, Use pattern Singleton and recalculate then you need that by method of your class.